cmd/aperturectl/cmd/apply/dynamic-config.go

```diff
@@ -60,61 +61,82 @@ var ApplyDynamicConfigCmd = &cobra.Command{
 			return fmt.Errorf("failed to connect to Kubernetes: %w", err)
 		}
 
-		c, err := k8sclient.New(kubeRestConfig, k8sclient.Options{
-			Scheme: scheme.Scheme,
-		})
-		if err != nil {
-			return fmt.Errorf("failed to create Kubernetes client: %w", err)
-		}
-
-		deployment, err := utils.GetControllerDeployment(kubeRestConfig, controllerNs)
-		if err != nil {
-			return err
-		}
-
 		dynamicConfigYAML := make(map[string]interface{})
 		err = yaml.Unmarshal(dynamicConfigBytes, &dynamicConfigYAML)
 		if err != nil {
 			return fmt.Errorf("failed to parse DynamicConfig YAML: %w", err)
 		}
-		dynamicConfigBytes, err := json.Marshal(dynamicConfigYAML)
-		if err != nil {
-			return fmt.Errorf("failed to parse DynamicConfig JSON: %w", err)
-		}
 
-		policy := &policyv1alpha1.Policy{}
-		err = c.Get(context.Background(), k8sclient.ObjectKey{
-			Namespace: deployment.Namespace,
-			Name:      policyName,
-		}, policy)
-		if err != nil {
-			if apimeta.IsNoMatchError(err) {
-				var dynamicConfigStruct *structpb.Struct
-				dynamicConfigStruct, err = structpb.NewStruct(dynamicConfigYAML)
-				if err != nil {
-					return fmt.Errorf("failed to parse DynamicConfig Struct: %w", err)
-				}
-				request := languagev1.PostDynamicConfigRequest{
-					PolicyName:    policyName,
-					DynamicConfig: dynamicConfigStruct,
+		if Controller.IsKube() {
+			var kubeClient k8sclient.Client
+			kubeClient, err = k8sclient.New(kubeRestConfig, k8sclient.Options{
+				Scheme: scheme.Scheme,
+			})
+			if err != nil {
+				return fmt.Errorf("failed to create Kubernetes client: %w", err)
+			}
+
+			var deployment *appsv1.Deployment
+			deployment, err = utils.GetControllerDeployment(kubeRestConfig, controllerNs)
```

Comment:
Nit: not sure why deployment became var?

Reply:
golangci-lint was complaining about err being defined above so had to do this to replave `:=` with `=`